shared/src/k8s/service.rs

```diff
@@ -136,6 +136,10 @@ pub fn create_new_service_from_spec(
             AKRI_INSTANCE_LABEL_NAME.to_string(),
             instance_name.to_string(),
         );
+        labels.insert(
+            AKRI_CONFIGURATION_LABEL_NAME.to_string(),
```

Comment:
somewhere, we test what labels get added, that test should be updated to validate this